11.MovieDataApi.js

```diff
@@ -1,10 +1,15 @@
 var express=require('express');
 var app=express();
 var request=require('request');
+homePage = '11Movie.ejs';
+resultPage = '11MovieResult.ejs';
+staticPath = "static";
+
+app.use(express.static(staticPath));
```

Comment:
Good changes, but due to some reasons this server is not fetching the results. I did'nt understood what this staticPath will do. but can you figure out why is this not working and make changes t the server (or keep it original)

Reply:
I also think it wold be great if you have added styles to the results page also. you can choose boostrap also if css is time consuming.